src/app/features/register-student/register-student.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="container">
+    <div class="register-form-students">
+        <form class="form-students" [formGroup]="formRegisterStudent" (ngSubmit)="
+        Send(formRegisterStudent.value)">
```

Comment:
use camelCase starting with lowercase in functions
```suggestion
        <form class="form-students"
            [formGroup]="formRegisterStudent"
            (ngSubmit)="Send(formRegisterStudent.value)">
```